core/templates/dev/head/pages/email_dashboard/email_dashboard_result.html

```diff
@@ -0,0 +1,73 @@
+{% extends 'pages/base.html' %}
+
+{% block maintitle %}
+  Email Dashboard Result - Oppia
+{% endblock maintitle %}
+
+{% block header_js %}
+  {{ super() }}
+  <style>
+    html, body {
+      background-color: #f2f2ee;
+    }
+    .success {
+      color: #0000ff;
+    }
+    .oppia-email-form {
+      text-align: center;
+      margin-top: 2%;
+    }
+    textarea {
+      margin-bottom: 1%;
+      border-radius: 10px;
+    }
+    .invalid-input {
+      color: red;
+    }
+    .success {
+      color: green;
+    }
+  </style>
+{% endblock header_js %}
+
+{% block navbar_breadcrumb %}
+  <ul class="nav navbar-nav oppia-navbar-breadcrumb">
+    <li>
+      <span class="oppia-navbar-breadcrumb-separator"></span>
+      Email Dashboard
+    </li>
+  </ul>
+{% endblock navbar_breadcrumb %}
+
+{% block content %}
+  <div class="oppia-email-form" ng-controller="EmailDashboardResult">
+    <p ng-if="emailSubmitted" class="success">Emails sent successfully. You will be redirected to Email Dashboard page.</p>
+    <p ng-if="invalid.subject" class="invalid-input">Please enter valid email subject</p>
+    <p ng-if="invalid.body" class="invalid-input">Please enter valid email body</p>
+    <p ng-if="invalid.num_users" class="invalid-input">Please enter valid no. of recipients</p>
+    <form class="form" ng-submit="submitEmail()">
+      Email subject: <textarea rows="2" cols="10" placeholder="Enter email subject here..." ng-model="email_subject"></textarea>
+      <br>
+      Email body: <textarea rows="10" cols="20" placeholder="Enter email body here..." ng-model="email_body"></textarea>
+      <br>
+      Email intent: <select ng-model="email_intent" ng-options="intent for intent in possible_email_intents"></select><br><br>
+      Email option: <input type="radio" name="email" ng-model="email_option" value="all" checked> Send to all
+      <input type="radio" name="email" ng-model="email_option" value="custom"> Send to max N users:
```

Comment:
Ah, I see why the backend was the way it was. Don't let the UI force the design of your backend, though -- that's kind of the tail wagging the dog. If you need to do some transformations, do it in the JS here before sending stuff to the backend.